### DISCUSSION TITLE
Custon Log Messages

### DISCUSSION THREAD

**Discussion by erayymz** [Flexget/Flexget#3674] [Q&A]
Hello,

I have the following configuration. It is a simple input to watch RSS and output to send notification. I am running daemon with info logging. The following task runs fine as I am getting notifications via the pushover, however, there is no logs in the log file. I guess notify plugin does not generate logs. Is there a way I can force-log something from the confirmation file?

 4k-notify:
    rss: RSS ADDRESS
    regexp:
      accept:
        - ^(?=.*2022).*$: {from: title}
        - ^(?=.*2023).*$: {from: title}
        - ^(?=.*2024).*$: {from: title}
        - ^(?=.*2025).*$: {from: title}
    notify:
      entries:
        title: 4k-notify
        via:
          - pushover:
              user_key: KEY
              api_key: KEY

An alternative would be adding the following in notify.py on lines 136, and 155 looks like. I rather inject logs via config file though.
logger.info('Notification Sent!')

**Comment by BrutuZ** [comment]
But, it does? Just make sure your log is at the Verbose level
```
2023-01-31 14:07:46 INFO     download      test Downloading: lorem ipsum
2023-01-31 14:07:47 VERBOSE  details       test Summary - Accepted: 1 (Rejected: 0 Undecided: 0 Failed: 0)
2023-01-31 14:08:18 VERBOSE  notify        test Successfully sent a notification to `telegram`
2023-01-31 14:08:19 VERBOSE  notify        test Successfully sent a notification to `discord`
```

**Reply by erayymz** [reply]
Yes it does in verbose. If I set the log to verbose it was too much. Anyways, just wanted to see if I can inject custom logs.
